### DISCUSSION TITLE
Not seeing the workloads tab

### DISCUSSION THREAD

**Discussion by dhrumil83** [microsoft/Microsoft-Fabric-workload-development-sample#32] [Q&A] [answered]
I am not seeing the workloads tab even after deployment to tenant and not able to do these steps: 
https://learn.microsoft.com/en-us/fabric/workload-development-kit/more-workloads-add

Any suggestion if this should work?

**Comment by ramizsha** [comment]
If you're refering to the workload hub - It's not available yet.
You should be able to see your workload nonetheless if you click on the workload switcher in the bottom left corner.

**Reply by ramizsha** [reply, marked as answer]
The workloads tab will be available once you publish your workload (see https://learn.microsoft.com/en-us/fabric/workload-development-kit/publish-workload-flow).
A workaround would be to add showWorkloadHubWithEmptyMarketplace=1 to the url as a parameter. (for example https://app.fabric.microsoft.com/home?experience=power-bi&showWorkloadHubWithEmptyMarketplace=1)
